src/components/Buttons/Button.d.ts

```diff
@@ -39,6 +39,13 @@ export interface ButtonProps
    * tag with the supplied href value.
    */
   href?: string
+
+  /**
+    * If passed, users will have the ability to render a custom element
```

Comment:
We should just describe the prop, not say what happens the user passes it.

Plus, if they pass it, they don't have the ability to decide,

> A custom element to render instead of a `button`. Will receive all props usually passed to the button element.

or something like that